src/objects.rs

```diff
@@ -1528,6 +1528,20 @@ impl AddPrefix for Transfer {
     }
 }
 
+impl Hash for Transfer {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        (&self.from_stop_id, &self.to_stop_id).hash(state);
+    }
+}
+
+impl PartialEq for Transfer {
+    fn eq(&self, other: &Transfer) -> bool {
+        self.from_stop_id == other.from_stop_id && self.to_stop_id == other.to_stop_id
+    }
+}
+
+impl Eq for Transfer {}
```

Comment:
Since we already have the `derivative` dependency in our dependencies, instead of manually implementing these, you can probably use `derivative` instead (see [documentation](https://mcarton.github.io/rust-derivative/latest/Hash.html)).